js/src/plugins/ajaxec.plugin.js

```diff
@@ -1,4 +1,4 @@
-import atkPlugin from 'plugins/atkPlugin';
+import atkPlugin from './atk.plugin';
```

Comment:
why atk is not is plugins ? I'm not saying to move it, just curious.

Reply:
not sure what you mean here. atk.plugin.js is the base class for all other plugins. ex: class ajaxec extends atkPlugin...

Reply:
gotcha!